projects/client/src/lib/dialog/dialog.component.ts

```diff
@@ -0,0 +1,140 @@
+import { Component, Input, ViewChild } from '@angular/core';
+import { MatDialogRef } from '@angular/material';
+import { Type } from '@tstack/core';
+import { Observable } from 'rxjs';
+
+import { TskDynamicContentComponent } from '../dynamic-content/dynamic-content.component';
+import { TskOption } from '../option/option';
+
+import { TskDialogActionButton } from './dialog-action-button';
+import { TskDialogConfig } from './dialog-config';
+import { TskDialogType } from './dialog-type';
+
+@Component({
+	selector: 'tsk-dialog',
+	templateUrl: './dialog.component.html',
+	styleUrls: ['./dialog.component.scss']
+})
+export class TskDialogComponent<ContentT = any, ResultT = any> {
+	@Input() title: string;
```

Comment:
Remove `Input` decorator or set the full component up to be used outside of a dialog